rdkcentral/Lightning-UI-Components
# fix: updates to optimize tree shaking

## Description

This PR eliminates all re-exports from the codebase and replaces them with direct file references. The primary goal of this change is to improve our project's compatibility with tree shaking. By directly referring to files, we can ensure that only the code actually used in the project is included in the final bundle, which will lead to smaller bundle sizes and better performance in production environments.

## References

NO-JIRA

## Testing

Testing if tree shaking is working properly takes a few steps. You will need to utilize an existing or new project to see the results.

1. Ensure Correct Webpack Configuration: Check that your Webpack configuration is optimized for tree shaking. This includes:

- Utilizing ES6 module syntax (import and export) throughout your project.
- Setting mode to 'production' in your Webpack configuration to enable optimizations like tree shaking.
- Confirming that the sideEffects property in your package.json is properly configured to false or accurately lists fles/modules that do have side effects, aiding Webpack in excluding unused modules.

2. Build the Project: Execute the Webpack build process using npm run build or your project's equivalent script to generate the production bundle. 

3. Review Output: After the build, review the output to ensure there are no errors or warnings related to the tree shaking process or the direct file references. This step is crucial to verify that the modifications have not introduced any build issues. The final bundle should only include Lightning UI components that have been used in your project. All others should no longer be present in the final code.

## Automation

All automation tests should pass as expected. This should have no effect on exported components or functions.

## Checklist
